prql-compiler/src/sql/std.sql.prql

```diff
@@ -38,9 +38,15 @@ let read_parquet = source -> s"read_parquet({source})"
 let read_csv = source -> s"read_csv_auto({source})"
 
 let mul = l r -> null
-let div_i = l r -> s"FLOOR(FLOOR(ABS({l} / {r})) * SIGN({l} / {r}))"
-let div_f = l r -> s"({l} * 1.0 / {r})"
-let mod = l r -> s"{l} % {r}"
+# If we want to raise an error when division by zero occurs, we can use the
+# following:
+# let div_i = l r -> s"CASE WHEN {l} * {r} < 0 THEN -FLOOR(ABS({l} / {r})) ELSE FLOOR(ABS({l} / {r})) END"
```

Comment:
If r = 0, there is still division by zero in the else branch.

Let's not concern ourselves with division by zero for now.